vignettes/vcr.Rmd

```diff
@@ -28,13 +28,18 @@ vcr introduction
 
 `vcr` is an R port of the Ruby gem [VCR](https://github.com/vcr/vcr) (i.e., a translation, there's no Ruby here :))
 
-`vcr` helps you stub HTTP requests so you don't have to repeat HTTP requests.
+`vcr` helps you stub and record HTTP requests so you don't have to repeat HTTP requests.
 
 The main use case is for unit tests, but you can use it outside of the unit test use case.
 
-`vcr` works with the `crul` and `httr` HTTP request packages; we're working on support for `curl`.
```

Comment:
didn't you drop this idea? It can be confusing for users cf https://github.com/ropensci/software-review/issues/329#issuecomment-723827982

Reply:
yes, curl support will not happen